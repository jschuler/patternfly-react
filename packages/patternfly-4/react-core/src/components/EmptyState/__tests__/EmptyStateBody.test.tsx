// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { EmptyStateBody } from '../EmptyStateBody';
// any missing imports can usually be resolved by adding them here
import {  } from '..';

describe('EmptyStateBody test', () => {
  it('EmptyStateBody should match snapshot', () => {
    const view = shallow(
      <EmptyStateBody
        children={<div>ReactNode</div>}
				className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
