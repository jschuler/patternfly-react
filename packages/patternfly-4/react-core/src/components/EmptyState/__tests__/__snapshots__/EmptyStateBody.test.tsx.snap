// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyStateBody test EmptyStateBody should match snapshot 1`] = `
<div
  className="pf-c-empty-state__body ''"
/>
`;
